### PR TITLE
{Compute} Update description of license_msg

### PR DESCRIPTION
**Pull request by BigCat20196**
**Description**

Resolve [#16541](https://github.com/Azure/azure-cli/issues/16541)

[Compute] Modify description of license_ msg. After reminding the user to set the license type on VMSS, execute the following command to apply the change to the existing VM.
`az vmss update-instances -g "Your resource group name" -n "Your VMSS name" --instance-ids "*" `

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by qwordy** [comment]
I would suggest update help in command level instead of parameter level

File: src/azure-cli/azure/cli/command_modules/vm/_params.py [hunk 1]
```diff
@@ -833,6 +833,8 @@ def load_arguments(self, _):
             license_msg = "Specifies that the Windows image or disk was licensed on-premises. " \
                           "To enable Azure Hybrid Benefit for Windows Server, use 'Windows_Server'. " \
                           "To enable Multitenant Hosting Rights for Windows 10, use 'Windows_Client'. " \
+                          "Remember that after setting the license type on VMSs, execute the following command to apply the changes to the existing VM." \
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/vm/_params.py:836]:
```suggestion
                          "Remember that after setting the license type on VMSS, execute the following command to apply the changes to the existing VM." \
```

File: src/azure-cli/azure/cli/command_modules/vm/_help.py [hunk 2]
```diff
@@ -2758,6 +2758,7 @@
 helps['vmss update'] = """
 type: command
 short-summary: Update a VMSS.
+long-summary: To update a global scale set property, you must update the property in the scale set model.Remember to execute the `az vmss update-instances -g 'Your resource group name' -n 'Your VMSS name' --instance-ids '*' ` command after using this command to make the change take effect. If you don't set the automatic modes for the upgrade policy.
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/vm/_help.py:2761]: Put it to short summary. Long summary is not shown in references.

Comment (BigCat20196) [reply src/azure-cli/azure/cli/command_modules/vm/_help.py]: OK，I've tried to write it in the short summary.

File: src/azure-cli/azure/cli/command_modules/vm/_help.py [hunk 3]
```diff
@@ -2757,8 +2757,7 @@
 
 helps['vmss update'] = """
 type: command
-short-summary: Update a VMSS.
-long-summary: To update a global scale set property, you must update the property in the scale set model.Remember to execute the `az vmss update-instances -g 'Your resource group name' -n 'Your VMSS name' --instance-ids '*' ` command after using this command to make the change take effect. If you don't set the automatic modes for the upgrade policy.
+short-summary: Update a VMSS.After that,remember to execute the 'az vmss update-instances' command to make the changes take effect, assuming that you have not set the automatic modes for the upgrade policy
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/vm/_help.py]:
```suggestion
short-summary: Update a VMSS. Run 'az vmss update-instances' command to roll out the changes to VMs if you have not configured upgrade policy.
```

File: src/azure-cli/azure/cli/command_modules/vm/_help.py [hunk 4]
```diff
@@ -2757,7 +2757,7 @@
 
 helps['vmss update'] = """
 type: command
-short-summary: Update a VMSS.
+short-summary: Update a VMSS. After that, remember to execute the 'az vmss update-instances' command to make the changes take effect, assuming that you have not set the automatic modes for the upgrade policy.
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/vm/_help.py:2760]:
```suggestion
short-summary: Update a VMSS. Run 'az vmss update-instances' command to roll out the changes to VMs if you have not configured upgrade policy.
```

**Comment by qwordy** [comment]
@BigCat20196 You should have seen a "Commit suggestion" button on my comment. If it looks good, just click it.

**Review by qwordy — APPROVED** [review]

**Pull request merged**